requirements.txt

```diff
@@ -1,9 +1,13 @@
 asyncpg
-psycopg2
+psycopg2-binary
 elasticsearch
 aiopg
-pydantic
+pydantic<2
```

Comment:
Self Review:

There was an error with one of the files with pydantic v2 and the simplest thing to do was to downgrade for now.